app/src/main/java/org/fossasia/openevent/app/event/attendees/contract/IAttendeesPresenter.java

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.openevent.app.event.attendees.contract;
+
+public interface IAttendeesPresenter {
+
+    void attach();
+
+    void dettach();
+
+    void loadAttendees(boolean forceReload);
+
+    void loadAttendees();
+
+    void toggleAttendeeCheckStatus(int position, Long attendeeId);
+
+    IAttendeesView getView();
```

Comment:
Presenter should not declare `getView` as view does not need to call it. Add it as a public method in presenter implementation because test needs it, but not in the contract itself